src/shared/ethtool-util.h

```diff
@@ -57,30 +57,23 @@ struct ethtool_link_usettings {
         } link_modes;
 };
 
+typedef struct u32_with_set {
+        uint32_t sz; /* a value of 0 indicates the hardware advertised maximum should be used.*/
```

Comment:
I'd call this `value` since this will be very useful for the coalescing stuff as well and not everything there is a size.